PackageExplorer/Models/NuGetFrameworkModel.cs

```diff
@@ -0,0 +1,255 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace NuGet.Frameworks
+{
+	/// <summary>
+	/// Class NuGetFrameworkModel.
+	/// </summary>
+	[PropertyChanged.ImplementPropertyChanged]
+	public class NuGetFrameworkModel
+	{
+		/// <summary>
+		/// Gets the instance.
+		/// </summary>
+		/// <value>The instance.</value>
+		public static NuGetFrameworkModel Instance { get; } = new NuGetFrameworkModel();
+
+		/// <summary>
+		/// Gets or sets the frameworks.
+		/// </summary>
+		/// <value>The frameworks.</value>
+		public IEnumerable<NuGetFramework> Frameworks { get; set; } = new List<NuGetFramework>();
+
+		/// <summary>
+		/// Gets the portable frameworks.
+		/// </summary>
+		/// <value>The portable frameworks.</value>
+		public ObservableCollection<NuGetTargetFrameworkItem> PortableFrameworks { get; private set; } = new ObservableCollection<NuGetTargetFrameworkItem>();
+
+		/// <summary>
+		/// Gets the selected frameworks.
+		/// </summary>
+		/// <value>The selected frameworks.</value>
+		public IEnumerable<NuGetFramework> SelectedFrameworks
+		{
+			get
+			{
+				var selectedPlatforms = (from platform in PortableFrameworks
+										 where platform.IsSelected
+										 select platform.SelectedItem?.Framework ?? platform?.Framework);
+
+				return selectedPlatforms;
```

Comment:
On other code parts Linq Extensions are used instead of the query syntax. I also prefer that. Can you rewrite that to 

`PortableFrameworks.Where(...).Select(..)` ? (Reshaper can do that for your)